mapeditor/Animation.h

```diff
@@ -83,6 +83,8 @@ class Animation
 	void load  (size_t frame, size_t group = 0);
 	void unload(size_t frame, size_t group = 0);
 
+	void exportBitmaps(const boost::filesystem::path& path, bool prependResourceName = false) const;
```

Comment:
it's never used with prependResourceName=False

Reply:
Same function as in CAnimation.h At one point they should be the same.

Reply:
Removed parameter